From b28bb856ccebe8fded51d11362bf5920699eed59 Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Thu, 18 Mar 2021 10:52:13 +0000
Subject: [PATCH] configure.ac: do not use compute-symver-floor

This script (newly introduced in 4.4.18) sporadically fails, without
failing configure(), which has been reported at
https://github.com/besser82/libxcrypt/issues/123

Upstream-Status: Inappropriate
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 configure.ac | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/configure.ac b/configure.ac
index 45b91b4..f3ba945 100644
--- a/configure.ac
+++ b/configure.ac
@@ -426,11 +426,7 @@ if test x$enable_shared != xyes; then enable_obsolete_api=0; fi
 # need to be binary backward compatible with.
 if test $enable_obsolete_api = 1; then
   AC_MSG_CHECKING([minimum symbol version to use for compatibility symbols])
-  SYMVER_FLOOR=$(
-    $PERL ${srcdir}/build-aux/compute-symver-floor \
-      ${srcdir}/lib/libcrypt.minver $host_os $host_cpu \
-    2>&AS_MESSAGE_LOG_FD
-  )
+  SYMVER_FLOOR=GLIBC_2.0
   AC_MSG_RESULT([$SYMVER_FLOOR])
   case "$SYMVER_FLOOR" in
     ERROR)
